codec/decoder/core/arm/intra_pred_neon.S

```diff
@@ -126,6 +126,7 @@ WELS_ASM_FUNC_END
 
 
 
+.align 3
```

Comment:
why the number is 3?

Reply:
Because `.align` on arm denotes the alignment as a power of two. This gives 8 byte alignment, which is needed as these are loaded 8 bytes all at once with vldr.